### PR TITLE
(#134) Create Proxy for Api V2

### PR DESCRIPTION
**Pull request by alerosad**
Closes (#134)

### CODE REVIEW

**Comment by alerosad** [comment]
Closes (#134)

File: src/index.js [hunk 1]
```diff
@@ -8,14 +8,15 @@ import ReactDOM from 'react-dom';
 
 import App from './App';
 import listingSupportApiFactory from './services/api/trial-listing-support-api';
-import clinicalTrialsSearchClientFactory from './services/api/clinical-trials-search-api';
+//import clinicalTrialsSearchClientFactory from './services/api/clinical-trials-search-api';
```
Comment (arcepaul) [inline src/index.js:11]: remove

File: src/services/api/clinical-trials-search-api/__tests__/getClinicalTrials.test.js [hunk 2]
```diff
@@ -35,8 +34,9 @@ describe('testing getClinicalTrials', () => {
 		});
 
 		const scope = nock('http://example.org')
-			.post('/clinical-trials', query.payload)
-			.reply(200, { total: 57, trials: [{}] });
+			.log(console.log)
```
Comment (arcepaul) [inline src/services/api/clinical-trials-search-api/__tests__/getClinicalTrials.test.js:37]: remove log

File: src/services/api/clinical-trials-search-api/getClinicalTrials.js [hunk 3]
```diff
@@ -5,14 +5,16 @@
  * @param {object} query the cts query
  */
 export const getClinicalTrials = async (client, query) => {
+	//var res = '';
```
Comment (arcepaul) [inline src/services/api/clinical-trials-search-api/getClinicalTrials.js:8]: remove

File: src/views/Disease/Disease.jsx [hunk 4]
```diff
@@ -67,12 +67,13 @@ const Disease = ({ routeParamMap, routePath, data }) => {
...
 					return {
 						...acQuery,
-						'primary_purpose.primary_purpose_code': paramData.idString,
+						primary_purpose: paramData.idString,
 					};
 				case 'interCodeOrPurl':
 					return {
 						...acQuery,
-						'arms.interventions.intervention_code': paramData.conceptId,
+						//'arms.interventions.intervention_code': paramData.conceptId,
```
Comment (arcepaul) [inline src/views/Disease/Disease.jsx:75]: remove

File: src/views/Intervention/Intervention.jsx [hunk 5]
```diff
@@ -59,12 +59,13 @@ const Intervention = ({ routeParamMap, routePath, data }) => {
 				case 'codeOrPurl':
 					return {
 						...acQuery,
-						'arms.interventions.intervention_code': paramData.conceptId,
+						//'arms.interventions.intervention_code': paramData.conceptId,
```
Comment (arcepaul) [inline src/views/Intervention/Intervention.jsx:62]: remove

**Comment by seyilonge-nci** [comment]
PR okay with me, once updates proposed by @arcepaul and @kate-mashkina are done. Just make sure to run formatter on files touched.

**Review by seyilonge-nci — COMMENTED** [review]
PR okay with me, once updates proposed by @arcepaul and @kate-mashkina are done. Just make sure to run formatter on files touched.

**Comment by alerosad** [comment]
will do everyone

**Comment by alerosad** [comment]
still working on getting the react-app-dev side working, currently it's a 200 when it makes the request to the api, but I get a CORS error

**Comment by alerosad** [comment]
Ok it's working now with the changes update from the api team

**Comment by alerosad** [comment]
https://react-app-dev.cancer.gov/clinical-trials-listing-app/pr-138/lung-cancer?cfg=0

**Comment by KateMashkinaNIH** [comment]
QA compared the number of results returned between PR and develop branch and as well trial's info
Results number is either same or very close, the order of trial results displayed is slightly different which is expected due to API update

**Comment by alerosad** [comment]
Paul's requested changes were made

**Comment by mworrest** [comment]
Passes UX review.

**Pull request merged**